packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -1451,6 +1468,15 @@ export const DEFAULT_GAS_SCHEDULE: Required<FeeSchedule> = {
         }
         return gas;
     },
+    [ERC20BridgeSource.Lido]: (fillData?: FillData) => {
+        const takerToken = (fillData as LidoFillData).takerToken;
+        // WETH avoids one wrapping and so it's slightly cheaper
+        if (takerToken === MAINNET_TOKENS.WETH) {
+            return 226e3;
+        }
+
+        return 255e3;
```

Comment:
IMHO this should be handled in `exchangeProxyOverhead` rather than here. It works out to be the same and the intent is the same, but the WETH MixinLido behaves the same regardless. It get's WETH and it withdraws into ETH. Regardless of whether the trade is ETH or WETH.

It's the `exchangeProxyOverhead` which introduces the ETH->WETH deposit overhead.

The way I think about it is, from the perspective of the `BridgeAdapter`, given the choice of two dexes, what is the cost difference? ETH->WETH deposit has already happened so the only option is to take the WETH and withdraw it.

This will be captured in SwapRevertSampling so it makes no matter, just thought I'd drop how I think about it